Tools/spec_coverage/README.md

```diff
@@ -0,0 +1,5 @@
+Command-line application that shows Dart specification coverage by co19 tests.
+
+Usage:
+`$ cd Tools/spec_coverage`
```

Comment:
Can we use `tools` instead? co19 is the only place we use uppercase in directories, and in the Dart SDK we have a `tools` directory as well. Preferable we'd change `Utils` as well. But we should leave the test directories alone because we don't want the churn of renaming tests.

Reply:
Renamed.